src/components/inputs/localized-multiline-text-input/translation-input.js

```diff
@@ -139,11 +139,11 @@ export default class TranslationInput extends React.Component {
         </div>
         <div className={styles.commandsContainer}>
           <div className={styles.commandsLeft}>
-            {(() => {
-              if (this.props.error) return <div>{this.props.error}</div>;
-              if (this.props.warning) return <div>{this.props.warning}</div>;
-              return this.props.languagesControl;
-            })()}
+            {do {
+              if (this.props.error) <div>{this.props.error}</div>;
+              else if (this.props.warning) <div>{this.props.warning}</div>;
+              else this.props.languagesControl;
+            }}
```

Comment:
At least now nobody needs to google: "IIFE in React" cause one fails every time to put the right amount of `(` and `{`s. :)